src/lib/paginator/paginator-intl.ts

```diff
@@ -48,7 +48,7 @@ export class MatPaginatorIntl {
 }
 
 /** @docs-private */
-export function MAT_PAGINATOR_INTL_PROVIDER_FACTORY(
+export function parentIntl(
```

Comment:
Sorry, I wasn't totally clear. I meant the argument to the function (so that it fits on one line):
```ts
export function MAT_PAGINATOR_INTL_PROVIDER_FACTORY(parentIntl: MatPaginatorIntl) {
  return parentMatPaginatorIntl || new MatPaginatorIntl();
}
```

Reply:
okay, will make the change, just to be sure with `parentMatPaginatorIntl` you mean the `parentIntl` argument correct?

Reply:
Also did you see my first comment, should pattern also be implemented for the `sort-header-intl`